quarkus-asyncapi/deployment/src/main/java/io/quarkiverse/asyncapi/generator/AsyncApiCodeGenerator.java

```diff
@@ -38,7 +38,7 @@ public AsyncApiCodeGenerator(Path outPath, Config config, Optional<String> packa
         this.outPath = outPath;
         this.config = config;
         this.basePackage = packageName
-                .orElse(config.getOptionalValue(AsyncApiConfigGroup.PACKAGE_PROP, String.class).orElse(DEFAULT_PACKAGE));
+                .orElse(config.getOptionalValue(AsyncApiConfig.PACKAGE_PROP, String.class).orElse(DEFAULT_PACKAGE));
```

Comment:
I rename the class as a `ConfigGroup` has a very special meaning in SmallRye Config and this is not a `ConfigGroup`, it's a root config.